EIPS/eip-draft-smartcontract-fee-share.md

```diff
@@ -0,0 +1,18 @@
+---
+eip: <to be assigned>
+title: Smart Contract Deployer Transactions Fee Share
+author: @tarekskr
```

Comment:
```suggestion
author: tarekskr (@tarekskr)
```
Handle should be in parenthesis. Please use the name to the left as your full name or moniker you'd like to officially be referred to as.